ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StyleFactory.kt

```diff
@@ -564,4 +565,12 @@ internal class StyleFactory(
             .orSuccessfullyNull()
             // Ensure the default source keyed by the default locale is present in the result.
             .map { nonEmptyMapOf(localizations.entry.key to this, it.orEmpty()) }
+
+    private fun Offering.getPackageOrNull(identifier: String): Package? =
+        @Suppress("SwallowedException")
+        try {
+            getPackage(identifier)
+        } catch (e: NoSuchElementException) {
```

Comment:
I think you can change to use `_` and it won't complain about the `SwallowedException`?
```suggestion
        } catch (_: NoSuchElementException) {
```

Reply:
TIL, thanks!